block_my_feedback.php

```diff
@@ -173,19 +173,19 @@ public function fetch_marking(stdClass $user): ?array {
      *
      * TODO - turnitin, quiz.
      *
-     * @param stdClass $mod
+     * @param cm_info $mod
      * @param stdClass $assess
      */
```

Comment:
Missing `@return`.

Reply:
return is specified in ?stdClass so no need to excessively document.